Formula/mongoose.rb

```diff
@@ -3,8 +3,7 @@ class Mongoose < Formula
   homepage "https://github.com/cesanta/mongoose"
   url "https://github.com/cesanta/mongoose/archive/6.17.tar.gz"
   sha256 "5bff3cc70bb2248cf87d06a3543f120f3b29b9368d25a7715443cb10612987cc"
-  # license ["GPL-2.0", "Cesanta"] - pending https://github.com/Homebrew/brew/pull/7953
-  license "GPL-2.0"
+  license ["GPL-2.0", "Cesanta"]
```

Comment:
`Casenta` isn't listed in the [SPDX License List](https://spdx.org/licenses/) so I don't think we can allow it (it will fail `brew audit`)